aip/general/0236.md

```diff
@@ -258,25 +264,25 @@ message PreviewPolicyExperimentRequest {
 - If the method is called on an experiment with the rules representing a no-op,
   then the system **must** preview the deletion of the live policy.
 
-#### stop
+#### stopPreview
```

Comment:
should "preview" become "startPreview" then? it feels weird to have one be prefixed with the state change while the other is not.

Reply:
I am totally ok with this but lets get @shwoodard 's thoughts